packages/p2p/src/components/floating-rate/floating-rate.scss

```diff
@@ -42,15 +42,11 @@
         border-color: transparent;
         padding: 0 0.2rem;
         color: inherit;
+
         @include mobile {
             position: relative;
             right: 9rem;
         }
-
-        &--symbol {
-            font-size: 1.4rem;
-            line-height: 1.5;
```

Comment:
I'm replacing these repeatedly used font CSS styles with Text component in Input as a better practice. QA checklist will include all occurences.